src/main/java/org/jabref/gui/linkedfile/DownloadLinkedFileAction.java

```diff
@@ -100,7 +100,7 @@ public void execute() {
 
         Optional<Path> targetDirectory = databaseContext.getFirstExistingFileDir(filePreferences);
         if (targetDirectory.isEmpty()) {
-            dialogService.showErrorDialogAndWait(Localization.lang("Download file"), Localization.lang("File directory is not set or does not exist!"));
+            dialogService.showErrorDialogAndWait(Localization.lang("Download file"), Localization.lang("File directory needs to be set or does not exist. Please save the library."));
```

Comment:
I the next line. (you can use \n)
Please save your library and check the preferences

Reply:
How would that look like in the localization properties file?
I tried:
File\ directory\ needs\ to\ be\ set\ or\ does\ not\ exist.\nPlease\ save\ your\ library\ and\ check\ the\ preferences.=File directory needs to be set or does not exist.\nPlease save your library and check the preferences.
But IntelliJ shows "no usages".

Reply:
Nm, got it! Had to use \n\

Reply:
Nm, that's the case with all the strings having \n. Ignoring.

Reply:
Run the localizationConsistensyTest, it will tell you if it's right or not